rasa/nlu/test.py

```diff
@@ -1515,6 +1517,8 @@ def run_evaluation(
             disable_plotting,
         )
 
+    telemetry.track_nlu_model_test(test_data)
```

Comment:
do we also want to track `perform_nlu_cross_validation`?

Reply:
we can add that later, don't think it is an important one right now